packages/meteor-gazelle-createdby/lib/tests/server.js

```diff
@@ -0,0 +1,36 @@
+Tinytest.add('Createdby - On create, records creator (also as updater)',
+                                                        function (test) {
+  // Arrange
```

Comment:
@daveyshindig Im flattered you followed this, but it was a joke for the couple of UTs I had it in. Its a silly pattern followed at work. You can go ahead and remove.